js/libs/keycloak-admin-client/src/resources/identityProviders.ts

```diff
@@ -116,10 +116,11 @@ export class IdentityProviders extends Resource<{ realm?: string }> {
   });
 
   public importFromUrl = this.makeRequest<
-    {
-      fromUrl: string;
-      providerId: string;
-    },
+    | {
+        fromUrl: string;
+        providerId: string;
+      }
+    | FormData,
```

Comment:
The `FormData` type can be anything. Are we doing this in other places as well? Seems kinda counterintuitive to go from a well typed API to a generic map of data.

Reply:
right, it's what the api expects a post with form data